### ISSUE TITLE
The placeholder will be stretched when enableRefresh is set to true

### ISSUE THREAD

**Issue by karnadii** [mchome/flutter_advanced_networkimage#46]
App force close when fail to load image and the Circular Progress indicator become ugly (stretched)

**Comment by karnadii** [comment]
the sample code is exactly the same as example

```dart
// using zooming widget & transitiontoimage widget
ZoomableWidget(
  minScale: 0.3,
  maxScale: 2.0,
  // default factor is 1.0, use 0.0 to disable boundary
  panLimit: 0.8,
  child: Container(
    child: TransitionToImage(
      AdvancedNetworkImage(url, timeoutDuration: Duration(minutes: 1)),
      // This is the default placeholder widget at loading status,
      // you can write your own widget with CustomPainter.
      placeholder: CircularProgressIndicator(),
      // This is default duration
      duration: Duration(milliseconds: 300),
    ),
  ),
)
```

it only happen when the internet connection is poor, try loading an image without internet and the loading indicator will stretch,
Error:
![Error](https://i.imgur.com/qBhC1PQ.png)
after this the app will force close
log ```
I/flutter (15925): Retry failed!
I/flutter (15925): Failed to load https://4.bp.blogspot.com/-p7TPjlQgnok/W97_SxpEUTI/AAAAAAAAMQM/rWLyaFW9gcYcc-T29u1MQBXAYXCARm2ngCLcBGAs/s1600/Q9.jpg.
I/flutter (15925): Retry failed!
I/flutter (15925): Failed to load https://2.bp.blogspot.com/-QUK2yvc6dqs/W97_O-SzMmI/AAAAAAAAMPg/rCG8sEVzfw8EYqkAOc5ALruZ895gnEBqQCLcBGAs/s1600/Q10.jpg.
Lost connection to device.
```
Normal
![Normal](https://i.imgur.com/S2T1pz4.png)

**Comment by mchome** [comment]
Fixed in `0.3.10` :)

**Comment by karnadii** [comment]
thanks :)
